functions/src/triggers/userCreate.ts

```diff
@@ -0,0 +1,16 @@
+import * as functions from 'firebase-functions'
+import * as jsonWebToken from 'jsonwebtoken'
+import { addTokenToUser } from '../db/addTokenToUser'
+import { assertJWTEnv } from '../utils/assertConfig'
+
+export const userCreate = functions.auth.user().onCreate(async (user) => {
+    const jwtKey = assertJWTEnv()
+
+    const newJWTToken = jsonWebToken.sign({ uid: user.uid }, jwtKey, {
```

Comment:
you can destructuring 

```suggestion
    const { uid } = user
    const newJWTToken = jsonWebToken.sign({ uid }, jwtKey, {
```

Reply:
I'm not sure one way is clearer than the other, and destructuring, while being efficient, also add more computing than direct access to the property.

Reply:
EDIT: agree if reused.